src/apis/downloadPdf.js

```diff
@@ -32,7 +53,12 @@ export default store => (includeAnnotations = true) => {
     return;
   }
 
-  downloadPdf(store.dispatch, {
-    includeAnnotations,
-  });
+  // legacy interface: instance.downloadPdf(true);
```

Comment:
should we remove this?

Reply:
oh wait. i think you are referring to the fact that we support the legacy API / interface as well?

Reply:
ok i see. hopefully we can deprecate the legacy interface in the future so the code is slightly easier to manage.